networkx/readwrite/json_graph/cytoscape.py

```diff
@@ -98,8 +98,8 @@ def cytoscape_graph(data, attrs=None):
 
     for d in data["elements"]["edges"]:
         edge_data = d["data"].copy()
-        sour = d["data"].pop("source")
-        targ = d["data"].pop("target")
+        sour = d["data"].get("source")
+        targ = d["data"].get("target")
```

Comment:
I agree we could use direct access, I just used "get" method because I thought it could be a good replacement for "pop", which returns a default value of None. I guess "source" and "target" is something we will always have in the edges?

Reply:
Yes -- if we don't have source and target we don't get the nodes for the edge so we should raise an exception. direct lookup will do this nicely. :}